composer.lock

```diff
@@ -710,31 +818,32 @@
                     "type": "tidelift"
                 }
             ],
-            "time": "2022-05-24T11:49:31+00:00"
+            "time": "2023-01-26T09:26:14+00:00"
         },
         {
             "name": "symfony/yaml",
-            "version": "v6.1.2",
+            "version": "v5.4.31",
```

Comment:
Regressing a major version seems...not good?

Reply:
Do we really care? It fits whatever given specs, that's all that really matters, yeah?

Reply:
```
$ composer why --tree symfony/yaml
symfony/yaml v5.4.31 Loads and dumps YAML files
└──drupal/coder 8.3.22 (requires symfony/yaml >=3.4.0)
   └──__root__ dev-main (requires drupal/coder ^8.3)
```

`drupal/coder` is the only thing requiring it, and appears to be rather (excessively?) permissive in what it accepts.